pcsx2/gui/MainFrame.cpp

```diff
@@ -48,8 +49,14 @@ wxMenu* MainEmuFrame::MakeStatesSubMenu(int baseid, int loadBackupId) const
 		wxMenuItem* m = mnuSubstates->Append(loadBackupId, _("Backup"));
 		m->Enable(false);
 	}
-
+	//  Implement custom hotkeys (F2) + (Shift + F2) with translatable string intact + not blank in GUI.
```

Comment:
Double space after `//` in a few spots like this, also you might want to leave out the specific keys from the comment just incase that changes in the future? Saying you are adding the hotkeys should be enough imo.

Reply:
The comments are used for placeholder and to easily find it in the code, they won't see it anyway.